docs/Release_notes.md

```diff
@@ -1,5 +1,10 @@
 V2X-Hub Release Notes
 ----------------------------
+Version 7.2.2, released May 9th, 2022
+--------------------------------------------------------
+V2X Hub release 7.2.2 includes added Hotfix for CARMACloud Plugin to prevent repeated TCM broadcast and fix timeout bugs:
+- Issue 364: Fix remove TCMs with the same request ids as TCR if receiving a duplicated TCR request.Fix time out logic that cause segmentation fault. Add config parameter for the thread sleep for.Add configuration parameter and logic to control the number of TCMs being broadcast.
```

Comment:
Issue 364: Removing existing TCMs when receiving multiple TCRs with same request ID.
Add time out logic  for each TCR upon repeatedly broadcast associated TCMs.
Fix time out logic that cause segmentation fault.
Add configuration parameter to update thread sleep time.
Add configuration parameter and logic to control the number times that TCMs can repeatedly broadcast.
For each acknowledgement it receives from CMV, it only removes one TCM from the list.